### PR TITLE
Update: fixed typos from previous PR of readme files

### PR DESCRIPTION
**Pull request by rhoads-zach**
@Ladicek FYI, fixed as per your suggestions. Extra comma in rest-http-README.adoc is intentional due to clause and appositive starting the sentence.

### CODE REVIEW

File: docs/topics/crud-README.adoc [hunk 1]
```diff
@@ -27,4 +27,4 @@ endif::[]
 = ${mission} - ${runtime} Booster
 
 
-IMPORTANT: While you can run and interact with this booster on our local host, it requires that you also have a database installed and configured. This booster runs best when deployed on OpenShift with a PostGres database. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:{guideURL}[${runtime} Runtime Guide].
\ No newline at end of file
+IMPORTANT: While you can run and interact with this booster on our local host, it requires that you also have a database installed and configured. This booster runs best when deployed on OpenShift with a PostGreSQL database. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:{guideURL}[${runtime} Runtime Guide].
```
Comment (Ladicek) [inline docs/topics/crud-README.adoc]: OK so historically, PostgreSQL is "Post Ingres", but the correct name is PostgreSQL now.

**Comment by Ladicek** [comment]
Re clause and apposi... eh..., I'll have to leave that up to you as a native speaker, but it looked weird to me :-)

**Comment by rhoads-zach** [comment]
Thanks @Ladicek, merging.

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/e9a0f05aade898089219a27c5c9a5c01b05f627a

**Pull request closed without merging**